gpiozero/output_devices.py

```diff
@@ -562,8 +562,12 @@ def __init__(
             raise GPIOPinMissing('red, green, and blue pins must be provided')
         LEDClass = PWMLED if pwm else LED
         super(RGBLED, self).__init__()
-        self._leds = tuple(LEDClass(pin, active_high) for pin in (red, green, blue))
-        self.value = initial_value
+        try:
+            self._leds = tuple(LEDClass(pin, active_high) for pin in (red, green, blue))
+            self.value = initial_value
+        except:
+            self.close()
+            raise
```

Comment:
Nice work Dave :-)  Another great example of how things need to get really complicated behind the scenes, to keep outward appearances easy-to-use.

I guess the specific change above can be un-done now?